### PR TITLE
Make TypeScripter work with .net45 and .net core

### PR DESCRIPTION
**Pull request by marbletravis**

### CODE REVIEW

**Review by hanksjosh — APPROVED** [review]
Travis is my hero!

File: TypeScripter.nuspec [hunk 1]
```diff
@@ -13,8 +13,10 @@
     </dependencies>
   </metadata>
 	<files>
-		<file src="bin/Release/TypeScripter.exe" target="content\tools"/>
-		<file src="bin/Release/DocoptNet.dll" target="content\tools"/>
-		<file src="TypeScripter.Attributes/bin/Release/TypeScripter.Attributes.dll" target="lib"/>
+		<file src="TypeScripter/bin/Release/net45/TypeScripter.exe" target="content\tools"/>
```
Comment (marbletravis) [inline TypeScripter.nuspec:16]: @AsaBeckstead @darrelbrown does this look reasonable?

**Review by hanksjosh — APPROVED** [review]
I like that there's more green now.

**Comment by marbletravis** [comment]
@darrelbrown  moving this to a beta branch and going to do some pre-release publishes. I think I will probably need more than one pre-release publish to get the package right...

**Pull request closed without merging**